bindgen/spec.yml

```diff
@@ -1244,6 +1244,7 @@ classes:
       all_users: std::vector<SharedSyncUser>
       sync_manager: SharedSyncManager
       subscribers_count: count_t
+      default_base_url: std::string_view
```

Comment:
The `properties` key in the spec declare _instance_ properties (implemented through instance methods on the C++ class), not static properties of the `App`: We currently don't have a way to expose static methods from C++ as static properties on the class, nor static properties directly.

Do you intent for SDKs to be able to set this value? I don't expect that to be the case, since it's a string_view. In fact, is there an SDK need to expose it at all?

If you want this to be exposed as a read-only static member of `App`, you'd need to conform to the style of the existing C++ code and express this as a static `default_base_url` function on `App`.

Reply:
Thank you @kraenhansen - I have updated `App::default_base_url` to be a static accessor function (e.g. `App::default_base_url()`) - the purpose of this is to (hopefully) provide the default value to the SDKs so they don't have to define their own default base URL value.

Reply:
Until an SDK actually expects to consume this, I would suggest simply leaving it out of the spec.yml.